src/ontology/fbbt-edit.obo

```diff
@@ -134356,6 +134387,16 @@ intersection_of: part_of FBbt:00005893 ! embryonic/larval peripheral nervous sys
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-09-27T16:42:39Z
 
+[Term]
+id: FBbt:00058260
+name: ostial cell
+def: "A specialized cardiomyocyte that forms an ostium in the wall of the dorsal vessel. There are two such cells per ostium, the ostium itself being formed by an opening between the two cells (Lehmacher et al., 2012; Rotstein and Paululat, 2016)." [FlyBase:FBrf0219200, FlyBase:FBrf0238157]
+subset: cur
+synonym: "ostia-forming cardiomyocyte" EXACT [FlyBase:FBrf0219200]
+is_a: FBbt:00007319 ! cardial cell
```

Comment:
I think this should be 'part of' some 'ostiolar valve' (and 'ostiolar valve should be 'part of' some 'dorsal vessel')